aspnetcore/performance/caching/middleware.md

```diff
@@ -33,6 +33,9 @@ Configure the application to use the middleware when processing requests. The sa
 
 The Response Caching Middleware only caches 200 (OK) server responses. Any other responses, including [error pages](xref:fundamentals/error-handling), will be ignored by the middleware.
 
+>[!WARNING]
+> Responses containing content for authenticated clients must be marked as not cacheable to prevent the middleware from storing and serving those responses. See [Conditions for caching](#conditions-for-caching) for details on how the middleware determine if a response is cacheable.
```

Comment:
@JunTaoLuo  FIXED in https://github.com/aspnet/Docs/commit/600c1afc4b6bdb2915eed80f9fdb8bb2fa731b17